util/math.h

```diff
@@ -92,18 +92,23 @@ inline int CountTrailingZeroBits(T v) {
 #endif
 }
 
-#if defined(_MSC_VER) && !defined(_M_X64)
```

Comment:
By not having ifdef _MSC_VER, we have code lines with emitted code but not covered by unit tests.

Reply:
Are we trying to appease the unit test LOC coverage or always conditionally compile away unused code?

For the former we can simply do:

```
#ifdef _MSC_VER
```

For the latter we'd need to do:

```
#if defined(_MSC_VER) && (!defined(HAVE_SSE42) || (!defined(_M_X64) && !defined(_M_IX86)))
```

I dropped this because the old code appeared to be leaning towards the latter and I didn't want to maintain it that way. If you are looking for the former benefit, I can add it.

Reply:
> can add the simple ifdef

Done.